src/main/webapp/script.js

```diff
@@ -0,0 +1,5 @@
+//Splits string containing poem into array of lines
+function splitPoem(poem) {
```

Comment:
Not sure if this is required in the client. The server is probably going to get the uploaded poem as a json, do the split and save it to datastore. Please sync with colin on the server poem upload endpoint that he is working on.